src/scene/sprite/Sprite.ts

```diff
@@ -243,25 +258,25 @@ export class Sprite extends Container implements View
         }
     }
 
-    private _updateBounds()
+    private _updateTrimmedBounds()
     {
-        updateQuadBounds(this._bounds, this._anchor, this._texture, 0);
+        updateQuadBounds(this._trimmedBounds, this._anchor, this._texture, 0);
     }
 
-    private _updateSourceBounds()
+    private _updateBounds()
     {
         const anchor = this._anchor;
         const texture = this._texture;
 
-        const sourceBounds = this._sourceBounds;
+        const bounds = this._bounds;
 
         const { width, height } = texture.orig;
 
-        sourceBounds.maxX = -anchor._x * width;
-        sourceBounds.minX = sourceBounds.maxX + width;
+        bounds.minX = -anchor._x * width;
+        bounds.maxX = bounds.minX + width;
 
-        sourceBounds.maxY = -anchor._y * height;
-        sourceBounds.minY = sourceBounds.maxY + height;
+        bounds.minY = -anchor._y * height;
+        bounds.maxY = bounds.minY + height;
```

Comment:
Also fixes a bug in `_updateBounds`: `minX`/`maxX` and `minY`/`maxY` have been swapped there created and inverted bound.